### ISSUE TITLE
TypeError: self.time is undefined on Angular 1.6.4 on clicking timepicker icon

### ISSUE THREAD

**Issue by saadqc** [classlinkinc/angular-material-time-picker#14]
It works fine on previous angular version. Breaking on version 1.6.4.

TypeError: self.time is undefined

Breaks on bold line below. 
```
switch (self.type) {
              case TYPE_HOURS:
                if (self.noMeridiem) {
                  for (var i = 1; i <= 23; i++)
                    self.steps.push(i);
                  self.steps.push(0);
                  **self.selected = self.time.getHours() || 0;**
                }
                else {
                  for (var i = 1; i <= 12; i++)
                    self.steps.push(i);
                    self.selected = self.time.getHours() || 0;
                    if (self.selected > 12) self.selected -= 12;
                }

```

**Comment by tothtibor9182** [comment]
Same problem here with 1.6.4 AngularJS

**Comment by saadqc** [comment]
@tothtibor9182 : did you try this ? https://github.com/classlinkinc/angular-material-time-picker/issues/10

I managed to resolve this somehow.

**Comment by Andreasbaaserud** [comment]
Fixed in release  1.0.7
